src/plugins/virtuals.js

```diff
@@ -159,20 +201,32 @@ module.exports = function (Bookshelf) {
     };
   });
 
-  function getVirtual(model, virtualName) {
+  function getVirtual(model, virtualName, params) {
     const { virtuals } = model;
     if (_.isObject(virtuals) && virtuals[virtualName]) {
-      return virtuals[virtualName].get ? virtuals[virtualName].get.call(model)
-        : virtuals[virtualName].call(model);
+      // params can be a value or an array
+      if(params instanceof Array){
+        return virtuals[virtualName].get ? virtuals[virtualName].get.call(model,...params)
+        : virtuals[virtualName].call(model,...params);
+      }else{
+        return virtuals[virtualName].get ? virtuals[virtualName].get.call(model,params)
+        : virtuals[virtualName].call(model,params);
```

Comment:
Some recommended cleanup here would be to make `virtualName` reference a variable. Maybe this on line 207:
```js
const { [virtualName]: virtual } = virtuals;
```

And then an example for the return value above:
```js
return virtual.get ? virtual.get.call(model, params) : virtual.call(model, params);
```